pkg/cluster/cluster.go

```diff
@@ -268,11 +271,14 @@ func (c *Cluster) run(stopC <-chan struct{}) {
 				c.status.Control()
 			}
 
-			running, pending, err := c.pollPods()
+			running, pending, readyPods, unreadyPods, err := c.pollPods()
 			if err != nil {
 				c.logger.Errorf("fail to poll pods: %v", err)
 				continue
 			}
+			c.status.ReadyPods = readyPods
```

Comment:
Probably we should rename this to ReadyMembers and UnreadyMembers in etcd context. Add comments saying that the string representation of the member is both etcd member name and the pod name.